Makefile

```diff
@@ -267,6 +267,9 @@ CGOFLAG_TSH ?= $(CGOFLAG)
 ELECTRON_BUILDER_ARCH_amd64 = x64
 ELECTRON_BUILDER_ARCH = $(or $(ELECTRON_BUILDER_ARCH_$(ARCH)),$(ARCH))
 
+KUBECTL_VERSION ?= $(shell go run ./build.assets/kubectl-version/main.go)
```

Comment:
Can you set this either as a target variable or in the recipe? Otherwise every target will attempt to evaluate this, even ones that don't need it like `print-version`.